tony-core/src/main/java/com/linkedin/tony/TonyApplicationMaster.java

```diff
@@ -137,10 +138,6 @@
   private AtomicInteger numRequestedContainers = new AtomicInteger();
   private Map<String, List<ContainerRequest>> jobTypeToContainerRequestsMap = new HashMap<>();
 
-  // allocationRequestIds are allocated to tasks sequentially. lastAllocationRequestId
-  // tracks the latest allocationRequestId allocated.
-  private long lastAllocationRequestId = 0;
```

Comment:
If this is gone, how do we track the allocation request now?

Reply:
We match by priority -- see `TonySession.getAndInitMatchingTaskByPriority()`.